pkg/ui/frontend/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```

Comment:
You can rename these for mobile web use when added to the spring board

Reply:
Fixed